openedx/features/course_experience/static/course_experience/js/CourseSock.js

```diff
@@ -57,8 +57,8 @@ export class CourseSock {  // eslint-disable-line import/prefer-default-export
 
       // Log open and close events
       const isOpening = $toggleActionButton.hasClass('active');
-      const logMessage = isOpening ? 'User opened the verification sock.'
-          : 'User closed the verification sock.';
+      const logMessage = isOpening ? 'edx.bi.course.sock.toggle_open'
+          : 'edx.bi.course.sock.toggle_close';
       Logger.log(
```

Comment:
We can also send the `Logger.log` events to GA, but they have to be explicitly whitelisted for them to work. Using `analytics.track` bypasses the need for whitelisting since it goes straight to Segment, AFAIK. I'm okay with switching these, but the ACs said to keep `Logger.log`.